storage/object/api-cli/combining-iam-and-object-storage.mdx

```diff
@@ -1,280 +1,149 @@
 
-Bucket policies operate at the bucket level, and enable you to state clearly who can perform different actions on which specific bucket.
+Bucket policies operate at the bucket level, and enable you to state clearly who can do what in each bucket.
 
-If you need to configure more granular access to your Object Storage buckets, you can combine both IAM and bucket policies. For example, if you must restrict access to specific actions you can perform on a bucket, or if you need the bucket to be available only to certain users, you can use a combination of IAM and bucket policies.
+To configure more granular access to your Object Storage buckets, you can combine both IAM and bucket policies.
 
-<Lightbox src="scaleway-bucket-policy-iam-table.webp" alt="" />
-
-<Message type="important">
-  Each bucket can have only one bucket policy.
-</Message>
-
-## Combining IAM and bucket policies
-
-In order to perform actions on a bucket, a principal (user or application) must be granted access via an IAM policy, and optionally via a bucket policy.
-
-For example, if a user wants to list the objects in a bucket, they must ensure that:
-
-    - the IAM policy explicitly allows the user to list Objects (for example, with an `ObjectStorageReadOnly` or a `ObjectStorageObjectRead` permission set specified in the policy)
-    - if there is no existing bucket policy, the action allowed by IAM is enough to grant them list access
-    - if there is a bucket policy, the list action is explicitly allowed in the policy
+## Bucket policies and IAM overview
 
-<Message type="important">
-  When using bucket and IAM policies concurrently, keep in mind that:
-    - Even if you only want to define the permissions available for bucket policies, you must first define IAM policy permissions. This is a necessary step of the process.
-    - When you create a Scaleway API key with IAM, you are prompted to specify the API key’s [preferred Project for Object Storage](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/). This API key will always use this Project when carrying out Object Storage actions via any API/CLI.
-    - The [preferred Project](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/) must always be the one of the bucket where you wish to perform an action.
-</Message>
+### IAM
 
-<Message type="note">
-  Before the [arrival of IAM](/identity-and-access-management/iam/reference-content/migration/) principals in a Scaleway bucket policy were defined by the `project_id`. Now they can either be specified by the `user_id` or the `application_id`.
-</Message>
+[Identity and Access Management (IAM)](/identity-and-access-management/iam/concepts/#iam) allows you to control and manage access to your resources. For users of Scaleway Object Storage, this means that besides configuring access via bucket policies, you can use IAM policies to configure permissions.
```

Comment:
We could say again here "at the project level". Even though it's written below, that's the main fact